Brewfile

```diff
@@ -3,8 +3,12 @@ tap "homebrew/cask"
 
 brew "kubernetes-cli"
 brew "kubernetes-helm"
-brew "hyperkit"
-brew "docker-machine-driver-hyperkit"
-
-cask "aws-vault"
-cask "minikube"
+if OS.mac?
+  brew "hyperkit"
+  brew "docker-machine-driver-hyperkit"
+  cask "aws-vault"
```

Comment:
aws-vault is not required for the use of gsp-local, should we remove this?

Reply:
I found it in [some other GSP docs](https://github.com/alphagov/gsp/search?q=aws-vault&unscoped_q=aws-vault), which is why I added it originally. I don't mind.

Reply:
Paul is right. It shouldn't be a requirement for local.

However, this repo is for everything, including local. I don't know if having single `Brewfile` or both `Brewfile.local` and `Brewfile` is what we want...

bikeshedding

Reply:
Single Brewfile, I say, else people would have to use with `brew bundle --file=Brewfile.local`.